webcomponents/bundle/src/main/bundle/toolnames.properties

```diff
@@ -4,3 +4,4 @@ grades=Grades
 tasks=Tasks
 announcements=Announcements
 forums=Discussions
+jumptotop_title=Jump to top
```

Comment:
Why have you added this translation to the toolnames bundle? It doesn't make sense here. You can just create a new bundle called jumptotop.properties and load that in.

Reply:
Creating a different property file for just one tag seemed unnecessary because it would add another min 5 files in that bundle. I suggest creating a general.properties file to accommodate line translation changes. Let me know what you think.

Reply:
It depends on whether you would expect components with significant translations to also pull in the general file. So, would you put things like done=Done, cancel=Cancel, save=Save in the general file? If so, I'd put that logic in sakai-element.js so a single i18n object is created, instead of managing multiple objects in the js components.

Reply:
Why would you get five files, minimum? Wouldn't it be just one minimum, jumptotop.properties?

Reply:
1. jumptotop.properties
2. jumptotop_tr_TR.properties
3. jumptotop_ro_RO.properties
4. jumptotop_fr_FR.properties